src/e2e/step-definitions/createEventSteps.js

```diff
@@ -0,0 +1,191 @@
+const { Before, Given, Then, When, And } = require("@cucumber/cucumber");
+const { EventsHubPage } = require("../pages/eventsHub.page.js");
+const { EventDetailPage } = require("../pages/eventDetails.page.js");
+const { RegistrationForm } = require("../pages/registrationForm.page.js");
+const { EventsDashboard } = require("../pages/eventsDashboard.page.js");
+const { BasicInfo } = require("../pages/basicInfo.page.js");
+const testData = require("../config/test-data/eventRegistration.json");
+const constants = require("../config/test-data/constants.js");
+const { expect } = require('@playwright/test');
+const { AdobeIdSigninPage } = require('@amwp/platform-ui-lib-adobe/lib/common/page-objects/adobeidsingin.page.js');
+const Logger = require('../common-utils/logger.js');
+const { SpeakersAndHosts } = require("../pages/speakersAndHosts.page.js");
+const logger = new Logger();
+
+Given('I am on the ECC dashboard page', async function () {
+  try {
+    this.page = new EventsDashboard();
+    await this.page.open();
+    logger.logInfo("Navigated to the Events Dashboard page successfully.")
+  } catch (error) {
+    logger.logError("Failed to open the Events Dashboard page:", error.message);
+    throw new Error("Could not navigate to the Events Dashboard page. Please check the URL or connectivity.");
+  }
+
+  try {
+    await this.page.clickSignIn();
+    logger.logInfo("Sign-in clicked successfully.");
+
+    this.context(EventDetailPage);
+    await this.page.isElementVisible(this.page.locators.signInEmailForm, timeout = 60000);
+    logger.logInfo("Sign-in required, proceeding with sign-in");
+
+    this.context(AdobeIdSigninPage);
+    await this.page.signIn(this.credentials.username, this.credentials.password);
```

Comment:
are we using same credentials to create event and register? if yes please use different users.

Reply:
Yes now, we are using src\e2e\config\test-data\eventCreation.json for event creation credentials and src\e2e\config\test-data\eventRegistration.json for event registration credentials